server/shared/types/dataset.ts

```diff
@@ -1,3 +1,5 @@
+import { index } from 'd3'
```

Comment:
do not import `index`. this is index signature https://www.typescriptlang.org/docs/handbook/2/objects.html#index-signatures